src/main/resources/schema.sql

```diff
@@ -1,50 +1,82 @@
-create table course (
-    id bigint generated by default as identity,
-    title varchar(255) not null,
-    creator_id bigint not null,
-    created_at timestamp not null,
+create table course
+(
+    id         bigint generated by default as identity,
+    title      varchar(255) not null,
+    creator_id bigint       not null,
+    created_at timestamp    not null,
     updated_at timestamp,
     primary key (id)
 );
 
-create table ns_user (
-    id bigint generated by default as identity,
-    user_id varchar(20) not null,
-    password varchar(20) not null,
-    name varchar(20) not null,
-    email varchar(50),
-    created_at timestamp not null,
-    updated_at timestamp,
+create table session
+(
+    id              bigint AUTO_INCREMENT(2),
+    course_id       bigint      not null,
+    start_date      date        not null,
+    end_date        date        not null,
+    price           int         not null,
+    state           varchar(10) not null,
+    type            varchar(10) not null,
+    user_count      int default 0,
+    max_user_count  int default 0,
+    image_size      int         not null,
+    image_extension varchar(10) not null,
+    image_width     int         not null,
+    image_height    int         not null,
+    primary key (id)
+);
+
+create table user_session
```

Comment:
코드에서는 sessionUser 라는 이름으로 사용되고 있으니 테이블명도 같은 형식이면 좋을 것 같아요
작성하신 erd에도 session_user 라고 되어 있어요 
```suggestion
    primary key (id)
);

create table session_user
```

Reply:
h2에서 SESSION_USER 이게 예약어 더라구요 ㅠㅠ 
session_users로 테이블명 수정하겠습니다~!